go-common.mk

```diff
@@ -178,7 +178,7 @@ $(GOTESTCOVERHTML): $(GOTESTCOVERRAW)
 	esac
 
 # Go executables
-$(_GO_ROOT_BUILD_TARGET): $(GOSRC)
+$(_GO_ROOT_BUILD_TARGET): $(GOSRC) go.mod
```

Comment:
Sorry if my previous comment was unclear.  You do need the previous change as well, you just also need these two changes so that all the build variants are covered